packages/graphql-transformers/src/computedFieldsWhereTrasform/computedFieldsWhereTransformation.ts

```diff
@@ -128,6 +132,6 @@ function getEntityTypeFromWhereInput(whereInputName) {
   return introspectionUtils.findTypeInIntrospection(openCrudIntrospection, whereInputName.replace('WhereInput', ''));
 }
 
-module.exports = {
+export default {
   transformComputedFieldsWhereArguments
```

Comment:
weird export but I understand you wanted to keep the same structure

Reply:
Yup, mostly backwards compatibility as written in the other comment.